katas/content/distinguishing_states/plus_minus/Verification.qs

```diff
@@ -0,0 +1,30 @@
+namespace Kata.Verification {
+    open Microsoft.Quantum.Katas;
+
+    operation StatePrep_IsQubitPlus(q : Qubit, state : Int) : Unit is Adj {
+        if state == 1 {
+            // convert |0⟩ to |+⟩
+            H(q);
+        } else {
+            // convert |0⟩ to |-⟩
+            X(q);
+            H(q);
+        }
+    }
+
+    @EntryPoint()
+    operation CheckSolution() : Bool {
+        let isCorrect = DistinguishTwoStates_SingleQubit(
+            StatePrep_IsQubitPlus,
+            Kata.IsQubitPlus,
+            ["|+⟩", "|-⟩"],
```

Comment:
Here the labels need to be switched compared to the task single_qubit_measurements\distinguish_plus_and_minus: the first label corresponds to the state that should be identified as false and the second label - to the state identified as true. (This gave me quite a pause, seeing solution "return false" claim that it misclassifies |-⟩ as |+⟩!)
```suggestion
            ["|-⟩", "|+⟩"],
```

Reply:
Sorry for missing that! Changing it right away